hack/quickcluster/setup-nfs-quickcluster.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+set -e
+
+trap 'last_command=$current_command; current_command=$BASH_COMMAND' DEBUG
+trap 'echo "\"${last_command}\" command completed with exit code $?."' EXIT
+
+ROOT="$(realpath -mq ${BASH_SOURCE[0]}/../../..)"
+
+export QUICKCLUSTERKEY=/tmp/quickcluster.key
+export NAMESPACE=openshift-nfs-storage
+PORT=2049
+
+if [ -z "$1" ]; then
+    echo "usage: setup-nfs-quickcluster.sh <upi host name>"
+    exit 1
+fi
+
+export REMOTE=$1
+QOCP="https://$(echo ${REMOTE} |sed s/upi-0/api/):6443"
+
+wget -q https://gitlab.cee.redhat.com/cee_ops/quicklab/raw/master/docs/quicklab.key -O $QUICKCLUSTERKEY
```

Comment:
this won't work since users now need to use their own private key

so perhaps we'll also need to add info to readme that user needs to add the key to auth agent, wdyt?

Reply:
Not sure what you mean. I have run this script as is on a freshly provisioned quickcluster and it works perfectly

Reply:
hmm that's strange, my ssh connection to the cluster did not work when I was using this script and I resolved it by using my own private key (that I generated and shared in rover) instead.

Reply:
Did you try this script or the quicklabs one? The command is different (quicklab user vs quickcluster)

Reply:
```
❯ ./hack/quickcluster/setup-nfs-quickcluster.sh upi-0.psturcrdu.lab.upshift.rdu2.redhat.com
```

Reply:
I'll try again once (or if) the cluster is back